src/guides/v2.3/config-guide/revert-split-database.md

```diff
@@ -0,0 +1,69 @@
+---
+group: configuration-guide
+title: Configure a single database from a split database
```

Comment:
I think "Migrate" or "Revert" would be a better verb. Whatever you decide, the file name should better align with the title.